prboom2/src/g_game.c

```diff
@@ -3065,6 +3066,7 @@ void G_InitNew(int skill, int episode, int map, dboolean prepare)
 
   //jff 4/16/98 force marks on automap cleared every new level start
   AM_clearMarks();
+  AM_clearTrail();
```

Comment:
Let's rather include the call to AM_clearTrail inside AM_clearMarks itself.

Reply:
I consider trails separate from marks